src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,51 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    // Test Case 구현
+    @Test
+    @DisplayName("요구사항1: set size 테스트")
+    void setSizeTest() {
+        //when
+        int size = numbers.size();
+
+        //then
+        assertThat(size).isEqualTo(3);
```

Comment:
다음처럼 검증할 수도 있다는 점 단순 참고차 말씀드립니다.
```suggestion
        assertThat(numbers).hasSize(3);
```